pdb2pqr/aa.py

```diff
@@ -200,23 +200,16 @@ def rebuild_tetrahedral(self, atomname):
                 hcoords = newcoords
                 cg1atom = self.get_atom("CG1")
                 cbatom = self.get_atom("CB")
-                if bondatom.name == "CD1":
+                if bondatom.name == "CD1" or bondatom.name != "CG2":
                     ang = util.dihedral(
                         cbatom.coords,
                         nextatom.coords,
                         bondatom.coords,
                         hcoords,
                     )
-                elif bondatom.name == "CG2":
-                    ang = util.dihedral(
-                        cg1atom.coords,
-                        nextatom.coords,
-                        bondatom.coords,
-                        hcoords,
-                    )
                 else:
                     ang = util.dihedral(
-                        cbatom.coords,
+                        cg1atom.coords,
```

Comment:
Are you sure of this change?

Reply:
Yes, it is a rewrite of the `if/elif/else` to just be an `if/else`. The first `else` is elevated to the `if` with the `or bondatom.name != "CG2"` clause. Then first `elif` becomes the second `else`.